source/fundamentals/csfle.txt

```diff
@@ -13,6 +13,9 @@ Client-Side Field Level Encryption
 Overview
 --------
 
+In this guide, you can learn how to install and use **Client-Side Field
+Level Encryption** in the MongoDB Java driver.
+
 Client-Side Field Level Encryption (CSFLE) allows you to encrypt
```

Comment:
```suggestion
In this guide, you can learn how to install and use **Client-Side Field
Level Encryption (CSFLE)** in the MongoDB Java driver.

CSFLE allows you to encrypt
```

Suggestion: I think the [first instance of the term should get the acronym](https://docs.mongodb.com/meta/style-guide/style/abbreviations/#abbreviations)

> Show the abbreviation in parentheses after the spelled-out term on the first appearance on the page.